app/talk/search-result.cjsx

```diff
@@ -24,14 +24,10 @@ module.exports = React.createClass
     comment = @props.data
     discussion = @discussionFromComment comment
     section = parseSection(comment.section)
+    [owner, name] = comment.project_slug?.split('/') or []
 
     <div className="talk-search-result talk-module">
       <CommentLink comment={comment}>{comment.discussion_title}</CommentLink>
       <Markdown content={comment.body} project={@props.project} />
-      {if section is 'zooniverse'
-        <DiscussionPreview {...@props} discussion={discussion} comment={comment} />
-      else
-        [owner, name] = comment.project_slug.split('/')
-        <DiscussionPreview {...@props} discussion={discussion} owner={owner} name={name} comment={comment} />
-      }
+      <DiscussionPreview {...@props} discussion={discussion} owner={owner} name={name} comment={comment} />
```

Comment:
Refactor that crept in from when I was tracking down a bug.

I can remove/resend this separately if we care.